video.py

```diff
@@ -32,13 +33,11 @@ def render(name):
 
     #Loading background
     background_clip = "backgrounds/" + random.choice(os.listdir("backgrounds"))
-    background = VideoFileClip(background_clip).fx(vfx.resize, height=resolution[1]).fx(vfx.loop, duration=image_clips.duration).set_position(("center","center"))
-    
-    # Composite all the components
-    composite = CompositeVideoClip([background,image_clips],resolution)
-    composite.audio = sound_clips
-    composite.duration = sound_clips.duration
-
-    # Render
-    composite.write_videofile(f'render/{name}.mp4',threads=4,fps=24)
+    background = VideoFileClip(background_clip).loop(n=None).set_duration(sound_clips.duration).resize((resolution[0],resolution[1]), PIL.Image.Resampling.LANCZOS) 

```

Comment:
Does this not stretch the video?

Reply:
Hmm, you can check out how it was rendered here [on this tiktok video](https://www.tiktok.com/@clogodesign/video/7326174174790176033). That was a test with this code that I submited I think.

Reply:
Small note: when I was trying to use your repo I commented out upload to tiktok code because it was not working on my machine but I don't know why. Maybe it was issue on my side.